desktop_version/src/Music.cpp

```diff
@@ -165,10 +165,12 @@ class SoundTrack
                     vlog_error("Unable to queue sound buffer");
                     return;
                 }
-                if (FAudioSourceVoice_Start(voices[i], 0, FAUDIO_COMMIT_NOW))
+                if (FAudioSourceVoice_Start(voices[i], 0, 144))
                 {
                     vlog_error("Unable to start voice processing");
                 }
+                FAudioVoice_SetVolume(voices[i], volume, 144);
```

Comment:
Anything blocking this from being above the Start()?

Reply:
not that i know of. Want me to move it?

Reply:
Definitely - if that doesn't break anything it eliminates the need for operation sets entirely.

Reply:
Alright, will move it up and see if Dav gets the issue

Reply:
No problems so far!